test/apiv2/47-subnet-pools.at

```diff
@@ -0,0 +1,29 @@
+# -*- sh -*-
+#
+# subnet pools conversion tests
+#
+
+# Create a temporary containers.conf with custom DefaultSubnetPools
+cat > $TESTS_DIR/containers.default_subnet_pools.conf <<EOF
```

Comment:
Sorry TESTS_DIR is the test/apiv2 dir which contains the other static config file, I just copied pastes it from another test.
Doing this here means we write to the test dir which means the next commit could pick up the file. Either stay with the tmpdir approach or just use a static file.